adapters/adkernelAdn/info.yaml

```diff
@@ -0,0 +1,2 @@
+maintainer:
```

Comment:
You can delete this file.

`static/bidder-info/adkernelAdn.yaml` is important. `adapters/adkernelAdn/info.yaml` isn't used for anything... so it's misleading to have.

Did you see some bad docs which made you think you needed this? If so let me know... I should update them.